composer/distributed/dist_strategy.py

```diff
@@ -31,7 +31,6 @@
     get_mixed_precision,
     set_custom_fsdp_module_kwargs,
 )
-from composer.distributed.shared_utils import add_fsdp_oom_hooks
```

Comment:
WIP since we need to remove the functions in `shared_utils.py` that we're not using (need to get some `mcli` stuff fixed before that)